suricata/update/parsers.py

```diff
@@ -141,6 +135,9 @@
     (("--offline",),
      {'action': 'store_true',
       'help': "Run offline using most recent cached rules"}),
+    (("-V", "--version"),
+     {'action': 'store_true', 'default': False,
+      'help': "Display version"}),
```

Comment:
I think its OK to move `-V` here, but the check for this flag where it prints the version and exits will need to be changed as its not longer in the global arguments.  For example, this command fails now:
```
./bin/suricata-update
```